<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_technician</class>
 <widget class="QWidget" name="add_technician">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>兼职技术人员</string>
  </property>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hours">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>工作小时：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hoursalary">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>时薪：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_name">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>95</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_hours">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>195</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_hoursalary">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>295</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>textEdit_name</tabstop>
  <tabstop>textEdit_hours</tabstop>
  <tabstop>textEdit_hoursalary</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
